src/iconsManifest/supportedExtensions.ts

```diff
@@ -2758,6 +2758,14 @@ export const extensions: IFileCollection = {
       languages: [languages.python],
       format: FileFormat.svg,
     },
+    {
+      icon: 'pyup',
+      extensions: ['.pyup'],
+      filenamesGlob: ['.pyup'],
```

Comment:
Don't use `*Glob` for just one filename. `*Glob` was designed to reduce lengthy declarations.

Reply:
Changed!